netgrasp/utils/mac.py

```diff
@@ -0,0 +1,255 @@
+from __future__ import print_function
+
+import os
+import re
+import datetime
+
+TEST_WSDATA = False
```

Comment:
Please add a comment, what is this? What is "WSDATA"? What's the significance of "TEST"?